pages/_/stats.js

```diff
@@ -0,0 +1,415 @@
+import * as React from "react";
+import * as Strings from "~/common/strings";
+import * as Constants from "~/common/constants";
+import * as SVG from "~/components/system/svg";
+import * as System from "~/components/system";
+import * as SchemaTable from "~/common/schema-table";
+
+import { css } from "@emotion/react";
+
+import Section from "~/components/core/Section";
+import ScenePage from "~/components/core/ScenePage";
+
+import ChartParent from "~/components/stats/ChartParent";
+
+const STYLES_ROW = css`
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  padding: 6px 24px;
+`;
+
+const STYLES_TEXT = css`
+  min-width: 5%;
+  padding-top: 6px;
+  width: 100%;
+`;
+
+const STYLES_SUBTEXT = css`
+  margin-top: 8px;
+  font-size: 12px;
+`;
+
+const STYLES_SUBTEXT_HIGHLIGHT = css`
+  margin-top: 8px;
+  font-size: 12px;
+  color: ${Constants.system.brand};
+`;
+
+const STYLES_ITEM = css`
+  margin-bottom: 12px;
+  display: inline-flex;
+  flex-direction: column;
+  max-width: 220px;
+  margin-right: 32px;
+`;
+
+const STYLES_ITEM_GROUP = css`
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+`;
+
+export default class StatsPage extends React.Component {
+  render() {
+    const EXAMPLE_MINERS = {
```

Comment:
Nit: This fixture shouldn't exist in render, shouldn't it be outside since you're not going to change it?

Reply:
Fixed, moved outside of render